_includes/footer.html

```diff
@@ -0,0 +1,11 @@
+<footer>
+	<div class="back-to-top tr pv5 pr5 back"><a href="" class="no-underline">Back to the top</a></div>
```

Comment:
I could be wrong but an empty href means no cursor pointer. Using `href="#"` adds it